app/screens/settings/notification_push/push_send.tsx

```diff
@@ -61,7 +61,7 @@ const MobileSendPush = ({sendPushNotifications, pushStatus, setMobilePushPref}:
             {sendPushNotifications &&
                 <>
                     <OptionItem
-                        action={setMobilePushPref}
+                        action={() => setMobilePushPref('all')}
```

Comment:
is this intended to be an inline function?, same with the others

Reply:
corrected